Source/ACE.Server/Managers/WorldManager.cs

```diff
@@ -193,6 +193,18 @@ private static void SendLoginRequestReject(IPEndPoint endPoint, CharacterError e
             tempSession.Network.Update();
         }
 
+        public static Session FindSessionByAccountId(uint AccountId)
```

Comment:
We alraedy have this defined as:

 public static Session Find(uint accountId)

Reply:
thanks, changed it to use `Find(`